modules/core/src/main/scala/sangria/execution/Executor.scala

```diff
@@ -19,7 +19,8 @@ case class Executor[Ctx, Root](
     deprecationTracker: DeprecationTracker = DeprecationTracker.empty,
     middleware: List[Middleware[Ctx]] = Nil,
     maxQueryDepth: Option[Int] = None,
-    queryReducers: List[QueryReducer[Ctx, _]] = Nil
+    queryReducers: List[QueryReducer[Ctx, _]] = Nil,
+    errorsLimit: Option[Int] = None
```

Comment:
The idea is to pass `errorsLimit` to `execute/prepare` methods and it will propagate it further to the query validator and to the variables validator.

Reply:
As an alternative, we could use the field `errorsLimit` on `queryValidator`.
I tried it quickly in https://github.com/bc-dima-pasieka/sangria/pull/1

Reply:
Yeah, that's how I implemented it initially https://github.com/sangria-graphql/sangria/pull/1022, but it seemed a bit weird to use the property of queryValidator to validate something but query 🤔

Reply:
I understand. I'll check again in the light of this new knowledge.
I was also trying not to break binary compatibility once again.

Reply:
https://github.com/sangria-graphql/sangria/pull/1049